src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/SendNotificationService.java

```diff
@@ -298,28 +297,31 @@ private Map<String, String> buildPersonalisation(CaseDetails caseDetails, String
     public void notify(CaseDetails caseDetails) {
         CaseData caseData = caseDetails.getCaseData();
         String caseId = caseDetails.getCaseId();
-        Map<String, String> emailData = getEmailData(caseData, caseId);
+        Map<String, String> emailData = getEmailData(caseData);
+
+        // include the link to all documents in the email to claimant
+        emailData.put(EXUI_ALL_DOCUMENTS_LINK, emailService.getExuiAllDocumentsLink(caseId));
```

Comment:
The email the claimant will get should go to citizen hub and not ExUI

Reply:
fixed: [38bba5a](https://github.com/hmcts/et-ccd-callbacks/pull/1834/commits/38bba5a041c26443bf88e40afb8a64864a81c0e1)